packages/opds-web-client/src/components/BasicAuthForm.tsx

```diff
@@ -23,19 +26,21 @@ export default class BasicAuthForm extends React.Component<BasicAuthFormProps, B
             { this.state.error }
           </div>
         }
+        <label htmlFor="login-input">{this.loginLabel()}</label>
```

Comment:
Removing the placeholders in favor of labels:

![Screen Shot 2019-11-07 at 3 28 15 PM](https://user-images.githubusercontent.com/1280564/68425067-4ba20e80-0173-11ea-8ada-4a29d2d02cf9.png)